src/test_logging.c

```diff
@@ -166,8 +167,23 @@ test_rstrnt_log_upload (gconstpointer user_data)
     g_assert_cmpint (message_callback_calls, ==, expected_calls);
 
     message_callback_calls = 0;
+}
+
+static void
+test_rstrnt_log_upload_no_logs (void)
```

Comment:
Would be nice to have some comments before your function describing what the test is trying to achieve.
So why "upload_no_logs"?  What was the cause?  due to Missing harness & task log?????

The same applies to other test functions.

Reply:
The name is self explanatory, test `rstrnt_log_upload ()` when there is no logs in the task. The same applies to other test functions.